Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM mcr.microsoft.com/playwright:v1.30.0-focal
 
-# Sets argument as environmental variable
-ENV PLAYWRIGHT_FOLDERNAME=checkout
+# Sets argument as environmental variable (default value)
```

Comment:
nit: mention the default value

Reply:
The default value is assigned in line 4, no need to add it as comment too I think